code/modules/surgery/organs/augments_internal.dm

```diff
@@ -8,17 +8,17 @@
 	var/implant_overlay
 	tough = 1 //not easyly broken by combat damage
 	sterile = 1 //not very germy
+	robotic = 2 // these are cybernetic after all
```

Comment:
It makes it possible to do robotic repair on a prosthetic if it has damage - which normally never happens outside of rare cases